buildtools/npm/manifest_test.go

```diff
@@ -0,0 +1,68 @@
+package npm
```

Comment:
This should be in `package npm_test` since you're not testing any private implementation details.